controllers/medusa/medusaconfiguration_controller_test.go

```diff
@@ -135,7 +135,7 @@ func testMedusaConfigurationKo(t *testing.T, ctx context.Context, f *framework.F
 		}
 		t.Logf("medusa configuration not ready yet")
 		return false
-	}, neverTimeout, interval)
+	}, timeout, interval)
```

Comment:
This adds mandatory 5s sleep always. You could just add time.Sleep(5 * time Second)